### PR TITLE
rspec model

### PR DESCRIPTION
**Pull request by thiephn-2606-zz**
Screenshot from 2021-08-19 07-38-28
redmine: https://edu-redmine.sun-asterisk.vn/issues/40378

### CODE REVIEW

**Comment by ducth-2598** [comment]
approve

**Comment by thiephn-2606-zz** [comment]
ready

File: training_system/spec/factories/tasks.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :task do
+    name { Faker::Name.name }
+    description {"bai tap ruby on rails"}
```
Comment (vanvtt-0952) [inline training_system/spec/factories/tasks.rb]: dùng faker đi cậu

File: training_system/spec/models/course_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    it { should validate_presence_of(:end_date) }
+
+    context "when tart date greater current date" do
+      it "should not be valid if start date greater current date" do
+        course_date.should_not be_valid
+      end
+    
+      it "raises an error if start date greater current date" do
+        course_date.valid?
+        course_date.errors.full_messages.should include("Start date Please select a start date greater than the current date")
```
Comment (vanvtt-0952) [inline training_system/spec/models/course_spec.rb]: `"Start date Please select a start date greater than the current date"` lấy từ I18n chứ cậu

**Comment by thiephn-2606-zz** [comment]
ready

File: training_system/app/models/course.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@ class Course < ApplicationRecord
   scope :search, ->(search){where("name LIKE ?", "%#{search}%")}
 
   # validates :course_subjects, presence: true
-  validates :user_courses, presence: true
+  # validates :user_courses, presence: true
```
Comment (vanvtt-0952) [inline training_system/app/models/course.rb]: Sao comment code rứa cậu

**Comment by thiephn-2606-zz** [comment]
ready

**Pull request merged**